### PR TITLE
MF-166 - Add lora-adapter service

### PR DESCRIPTION
**Pull request by manuio**
Signed-off-by: Manuel Imperiale <manuel.imperiale@gmail.com>

### CODE REVIEW

File: lora/message.go [hunk 1]
```diff
@@ -0,0 +1,43 @@
+package lora
+
+// RxInfo receiver parameters
+type RxInfo []struct {
```
Comment (drasko) [inline lora/message.go:4]: Why do we copy these? Can't we just import packages where these are defined from LoRa Server?

File: lora/api/logging.go [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+			lm.logger.Warn(fmt.Sprintf("%s with error: %s.", message, err))
+			return
+		}
+		lm.logger.Info(fmt.Sprintf("%s without errors.", message))
+	}(time.Now())
+
+	return lm.svc.MfxSubscribe()
+}
+
+func (lm loggingMiddleware) MfxToLoraProvision(es lora.EventSourcing) (err error) {
```
Comment (drasko) [inline lora/api/logging.go]: Just `Provision` or `LooraPrivision` is OK. We know how it works.

File: lora/api/logging.go [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+			lm.logger.Warn(fmt.Sprintf("%s with error: %s.", message, err))
+			return
+		}
+		lm.logger.Info(fmt.Sprintf("%s without errors.", message))
+	}(time.Now())
+
+	return lm.svc.LoraSubscribe()
+}
+
+func (lm loggingMiddleware) LoraToMfxRouter(m lora.Message) (err error) {
```
Comment (drasko) [inline lora/api/logging.go]: Just `Router` `MessageRouter` or `ChannelRouter` should be fine

File: lora/message.go (hunk 1 above)
Comment (manuio) [reply lora/message.go:4]: As we discuss, they are in `internal` lora app server repo. We keep this structures.

**Review by anovakovic01 — CHANGES_REQUESTED** [review]
Just fix these, and remove code that is commented out. After that, we can merge, and I will start working on client for ES.

**Review by anovakovic01 — APPROVED** [review]
LGTM!

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**